generatedComponentConfig.js

```diff
@@ -31,10 +31,10 @@ import NormalItemList from './packages/terra-clinical-detail-view/examples/test-
 import DefaultListItem from './packages/terra-clinical-detail-view/examples/test-examples/DetailListItem/DefaultListItem.example';
 import DetailViewDivided from './packages/terra-clinical-detail-view/examples/test-examples/DetailView/DetailViewDivided.example';
 import DetailViewNoDivider from './packages/terra-clinical-detail-view/examples/test-examples/DetailView/DetailViewNoDivider.example';
-import ButtonErrorView from './packages/terra-clinical-error-view/tests/nightwatch/ButtonErrorView';
```

Comment:
I know you already did this work, but I don't think we should move these file either. See conversation here https://github.com/cerner/terra-clinical/pull/296